sklearn/feature_selection/from_model.py

```diff
@@ -111,3 +139,115 @@ def transform(self, X, threshold=None):
             return X[:, mask]
         else:
             raise ValueError("Invalid threshold: all features are discarded.")
+
+
+class SelectFromModel(BaseEstimator, SelectorMixin):
+    """Meta-transformer for selecting features based on importance weights.
+
+    Parameters
+    ----------
+    estimator : object
+        The base estimator from which the transformer is built.
+        This can be both a fitted (if ``prefit`` is set to True)
+        or a non-fitted estimator.
+        If it a fitted estimator, then ``transform`` can be called directly,
+        otherwise train the model using ``fit`` and then ``transform`` to do
+        feature selection.
+
+    threshold : string, float, optional
+        The threshold value to use for feature selection. Features whose
+        importance is greater or equal are kept while the others are
+        discarded. If "median" (resp. "mean"), then the ``threshold`` value is
+        the median (resp. the mean) of the feature importances. A scaling
+        factor (e.g., "1.25*mean") may also be used. If None and if
+        available, the object attribute ``threshold`` is used. Otherwise,
+        "mean" is used by default.
+
+    prefit: bool, default True
+        Whether a prefit model can be passed or not.
```

Comment:
Actually my initial question was different, but now I really don't understand what you mean. (It might be just because I'm a bit sleepy).

The `threshold` parameter is not used at fit time (only at transform time) so how would it be able to do this selection?